recirq/quantum_chess/bit_utils.py

```diff
@@ -45,6 +45,16 @@ def num_ones(n: int) -> int:
         n = n // 2
     return count
 
+def bit_ones(n: int) -> List[int]:
+    """Indices of ones in the binary representation of n."""
+    indices = []
+    index = 0
+    while n > 0:
+        if n % 2 == 1:
+            indices.append(index)
+        n >>= 1
+        index += 1
+    return indices
```

Comment:
Wasn't this in your other PR?  Can you merge so that this is picked up?

Reply:
Hi Doug, yes this is strange. I merged that PR first, then created this one. Not sure why this is still showing up.

Reply:
Yes that's strange. I merged the last PR before committing this one for review. Not sure why this happens.

Reply:
I would try merging from master again.  The last merge from master was from 23 days ago, so that's probably why.

You should be able to do a 'git pull upstream master' in order to do this.

Can you do that before I review again?  Having all the extra stuff makes it more confusing to review.

Reply:
Thanks Doug. Now it's merged. Sorry for the confusion.